test/integration/test-one.sh

```diff
@@ -17,27 +17,28 @@ REPODIR=$(git rev-parse --show-toplevel)
 
 # Set and export the PATH to one that includes a go binary installed by the
 # Makefile, if necessary.
-PATH=$(cd "${REPODIR}"; make go-bin-path)
+PATH=$(cd "${REPODIR}" || exit; make go-bin-path)
```

Comment:
Will we get to see the error logged by `cd` here if we end up having to invoke `exit` ?

Reply:
Yup. `cd` will emit the failure (e.g. `bash: cd: whatever: No such file or directory`) to stderr.